### PR TITLE
Cna and putative driver coloring in expression tab

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/pages/resultsView/ResultsViewPage.tsx [hunk 1]
```diff
@@ -260,13 +260,14 @@ export default class ResultsViewPage extends React.Component<IResultsViewPagePro
 
                             const store = this.resultsViewPageStore;
 
-                            if (store.rnaSeqMolecularData.isComplete && store.studyIdToStudy.isComplete
-                                && store.mutations.isComplete && store.genes.isComplete && store.coverageInformation.isComplete) {
+                            if (store.studyIdToStudy.isComplete
+                                && store.putativeDriverAnnotatedMutations.isComplete && store.genes.isComplete && store.coverageInformation.isComplete) {
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPage.tsx]: putative driver contains rnaseq and mutations data?

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPage.tsx]: what?

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPage.tsx]: oh, no I just rewrote it to use numericGeneMolecularDataCache as a data source

**Comment by adamabeshouse** [comment]
- [ ] reverse tick label angle
- [ ] cbioTooltip class needs to be applied on Popover
- [ ] reduce right padding when no legend, but still leave room for angled tick labels
- [ ] move '* Driver annotations ... ' message outside borderedChart

**Pull request merged**